README.md

```diff
@@ -1917,6 +1918,14 @@ $ git profile use work
 ```
 Currently there will be no output in case of success
 
+## [git-exfiltrate](https://github.com/Incognito/git-exfiltrator)
+### Break apart large branches into smaller ones
+```
+git exfiltrate master feature-branch "b/*"
```

Comment:
Thanks for the PR! What does `git exfiltrate master feature-branch "b/*"` print and is the printed message informative enough so that you don't need to explain it above and below?

Reply:
I know this is 2 years later but I got back to this eventually. I added some details here.